pom.xml

```diff
@@ -22,6 +25,16 @@
   </dependencyManagement>
 
   <dependencies>
+      <dependency>
+          <groupId>javax.annotation</groupId>
+          <artifactId>javax.annotation-api</artifactId>
+          <version>1.3.1</version>
```

Comment:
A build property could be added here rather than hardcoding this but ... next protostream will not require this dependency so we'll remove it completely by ispn 12.final.